vernac/vernacentries.ml

```diff
@@ -2083,9 +2101,9 @@ let interp ?proof ~atts ~st c =
   | VernacGeneralizable gen -> vernac_generalizable ~atts gen
   | VernacSetOpacity qidl -> vernac_set_opacity ~atts qidl
   | VernacSetStrategy l -> vernac_set_strategy ~atts l
-  | VernacSetOption (key,v) -> vernac_set_option ~atts key v
-  | VernacSetAppendOption (key,v) -> vernac_set_append_option ~atts key v
-  | VernacUnsetOption key -> vernac_unset_option ~atts key
+  | VernacSetOption (export, key,v) -> vernac_set_option ~atts export key v
+  | VernacSetAppendOption (key,v) -> vernac_set_append_option ~atts false key v
```

Comment:
This is dead code. The constructor `VernacSetAppendOption` should be removed from the `vernac_expr` data type.